src/parser.ts

```diff
@@ -390,7 +395,11 @@ export class Parser {
                 const localDoc = await Parser.loadYaml(`${cwd}/${value.local}`);
                 includeDatas = includeDatas.concat(await Parser.prepareIncludes(localDoc, cwd, gitRemote, tabCompletionPhase));
             } else if (value["file"]) {
-                const fileDoc = await Parser.loadYaml(`${cwd}/.gitlab-ci-local/includes/${value["project"]}/${value["ref"] || "master"}/${value["file"]}`);
+                const fileDoc = await Parser.loadYaml(`${cwd}/.gitlab-ci-local/includes/${gitRemote.domain}/${value["project"]}/${value["ref"] || "master"}/${value["file"]}`);
```

Comment:
I think you gonna change "master" as default since this is being abandoned for "main". 
I think this is configurabale

Reply:
You can configure it from github under settings

Reply:
Gitlabs repo use master, so it doesn't matter... But what does that have to do with this line of code ?

Reply:
you default to master

Reply:
Yes, i've always done that. Got nothing to do with this particular feature @moberghammer